src/script/test-ceph-backport.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if git $current_pr stats | grep ceph-backport.sh ; then
+    git clone https://lab.fedeproxy.eu/ceph/ceph-backport
```

Comment:
This should be something hosted on ceph.com we control.

Reply:
@batrick I don't know much about it, how do we do that?

Reply:
I would ask @djgalloway to mirror it.